front-end/src/components/analysis-page/AnalysisPage.jsx

```diff
@@ -49,10 +49,29 @@ export class AnalysisPage extends Component {
             data-testid="analysis-page-main"
           >
             <div className="flex gap-y-10 w-full flex-col h-full">
-              <div className= "w-full bg-gray-100 pb-2">
+              <div className= "flow-root w-full bg-gray-100 pb-2">
+                {
+                    !this.state.showIntentMenu &&
+                    <nav>
+                      <button
+                        className="float-left m-auto px-5 py-2"
```

Comment:
```suggestion
                        className="float-left m-auto px-5 py-3"
```
To make it so that it's centered in the middle if you decide to commit my suggestion about the svg size